### PR TITLE
use tabular for orionBootstrapAccountsIndex

### PR DESCRIPTION
**Pull request by frabrunelle**
I've been wanting to make some improvements to Orion Accounts for a few weeks.

It's great that you have started to make some changes. I love what you did so far :smile: 

One feature that I really need is displaying the users in the Accounts index page with tabular so that even if I have hundreds (or thousands) of users, they don't all show up on one page. also the search feature is really useful.

I have other things I want to add but I need to think a bit more about what I need exactly.

to give some context, I am currently working on building a bitcoin brokerage software with Orion. The source is here: https://github.com/frabrunelle/satoshicounter

At the moment, I have created a separate collection for Members. this is useful because I want our employees to be able to sign up people who come at the [Bitcoin Embassy](http://bitcoinembassy.ca) to buy or sell bitcoins.

so for example, a client can provide his first name, last name, phone number and email. and we can use this information to create a new member and then create a trade associated with number of this new number. he doesn't have to provide us with a password and login into his own account. the employee can stay in his account and do the trade on behalf of the member.

now that you have added the option for admins to be able to create users (instead of simply inviting them), it would make sense for me to use the Meteor.users collection directly.

and it would be pretty cool because the client would receive an email which he can check later and set a password, but at least the employee can create the trade on his behalf while he is at our store.

one thing I would like is for the employee to be able to directly enter the information from the schema (firstName, lastName, phoneNumber and potentially other stuff) directly from the invite page.

on the other hand, I am thinking that it might make sense to have two pages: /accounts/create and /accounts/invite. let me know what you think and I will submit another pull request.

also, how about including the edit roles page inside the edit page? it doesn't seem necessary to have a separate page just for that? that way I could potentially remove the actions column (people just need to click on the row to access the edit page)

### CODE REVIEW

**Comment by mraak** [comment]
Tabular is in immediate pipeline. I'll review other comments when I'm back at the comp :)

**Comment by frabrunelle** [comment]
:+1:

had you already started implementing the Tabular integration? because this is exactly what this pull request is adding :smile:

**Comment by mraak** [comment]
> one thing I would like is for the employee to be able to directly enter the information from the schema (firstName, lastName, phoneNumber and potentially other stuff) directly from the invite page.

You can set the profile fields in lib/options.js file. There could also be an optional property for each of the fields "mandatoryForEnrolment" or something similar. Obviously when the user goes to set the password he'd also have to set the mandatory fields. Very nice to have.

> on the other hand, I am thinking that it might make sense to have two pages: /accounts/create and /accounts/invite. let me know what you think and I will submit another pull request.
> 
> also, how about including the edit roles page inside the edit page? it doesn't seem necessary to have a separate page just for that? that way I could potentially remove the actions column (people just need to click on the row to access the edit page)

For both above: There are already 3 forms on accounts edit page and adding 4th would definitely clutter the page. I'd leave it like this for now until someone comes up with a really good idea how the restructure the account edit page.

File: packages/accounts/accounts-tab/client.js [hunk 1]
```diff
@@ -4,23 +4,12 @@ ReactiveTemplates.onRendered('accounts.index', function() {
 })
 
 ReactiveTemplates.helpers('accounts.index', {
-  users: function () {
-    return Meteor.users.find({}, { sort: { createdAt: -1 } });
-  },
-  buttons: function() {
-    var self = this;
-    return _.filter(orion.accounts._adminUsersButtons, function(value, key, list){
-      if (typeof value.shouldShow != 'function') {
```
Comment (mraak) [inline packages/accounts/accounts-tab/client.js:13]: You deleted privileges for buttons display? Are you handling this elsewhere in a different manner?

Comment (frabrunelle) [inline packages/accounts/accounts-tab/client.js:13]: yes, that code has been moved just underneath :smile: (lines 24-35). It's exactly the same.

I had to put the code for the buttons in its own template in order to use it with Tabular.

Comment (mraak) [inline packages/accounts/accounts-tab/client.js:13]: Oh yes, got it. Will try and merge this today evening or tomorrow morning.

Comment (frabrunelle) [inline packages/accounts/accounts-tab/client.js:13]: I should probably just make a pull request directly to [orionjs/orion](https://github.com/orionjs/orion) (v1.1 branch) now that your PR has been merged. what do you think?

Comment (mraak) [inline packages/accounts/accounts-tab/client.js:13]: Yeah do that.

**Pull request closed without merging**